### ISSUE TITLE
Is there a possibilty of changing the graph timespan?

### ISSUE THREAD

**Issue by demahum** [oguzhaninan/Stacer#203]
When monitoring resources, e.g. memory usage, very nice graph is shown in real-time, but it is limited to last 60 seconds. Is it possible to make this value bigger? E.g. last 5 minutes or something like that.

**Comment by micwoj92** [comment]
Also related to this I think, possibility to set refresh time of graphs, many task managers can do that.

**Comment by PVLPM** [comment]
Yeah, bump this. This is very simple, but very important feature. Please, add!
It would be nice to enter any value in sec/min.
